test/bdd/fixtures/well-known-server/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+#
+# SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+version: '3'
+
+services:
+
+  static-file-server:
+    container_name: testnet.trustbloc.dev
```

Comment:
Don't we usually do trustbloc.local for dev machines and trustbloc.dev for actual deployment?

Reply:
done